### DISCUSSION TITLE
Multiple Filter Request

### DISCUSSION THREAD

**Discussion by NYIGCommunications** [code4recovery/12-step-meeting-list#808] [Q&A] [answered]
Hello, might it be possible to select multiple filters? For example, online + spanish, or,  in person + lgbt + literature

**Comment by joshreisner** [comment, marked as answer]
you can technically do this now, although it's currently only possible with a desktop keyboard. on mac you hold down the `command` key when selecting the second/third type (on windows i think it's either the windows key or the ctrl key)

here is in person + lgbt + literature: https://www.nyintergroup.org/meetings/?tsml-day=any&tsml-type=LGBTQ%2CLIT&tsml-attendance_option=in_person

**Reply by joshreisner** [reply]
one other thing that's less-than-ideal is that when there are no results it auto-clears the filters, rather than showing a 'no results' screen

**Reply by NYIGCommunications** [reply]
so cool, thank you, this helps lots!
